code/css/theme-default.css

```diff
@@ -0,0 +1,127 @@
+body {
+  background-color: black;
+}
+
+h1 {
+  text-align: center;
+  color: white;
+}
+
+/* box for entire calculator - needed to position the content */
+
+.rcalc-container {
+  text-align: center;
+  margin-bottom: 200px;
+}
+
+/* calculator form */
+
+.rcalc-calculator {
+  background-color: AliceBlue;
```

Comment:
Don't use for color names and then 6-digit format ... we will use just one type of color representing and this is 6-digit hexa  .. unify it on all places
https://www.google.cz/search?rlz=1C1GGRV_enCZ751CZ751&ei=1Dr5WsCdAciYsAGkgLNI&q=types+of+color+representing+css&oq=types+of+color+representing+css&gs_l=psy-ab.3...2316.3219.0.3403.4.4.0.0.0.0.124.461.0j4.4.0....0...1.1.64.psy-ab..0.2.232...33i22i29i30k1.0.TX74RKy7UIc

Reply:
6 digit format used only, eliminated color names in whole code